### DISCUSSION TITLE
Welcome to seait Discussions!

### DISCUSSION THREAD

**Discussion by diStyApps** [diStyApps/seait#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shoggot** [comment]
Maybe this should be an issue, but it's also possibly not SEAIT related, so...

When I attempt to install A1111, Vlad, UX, or Comfy (and possibly others, though these are what I've tried so far), I get:

```
Virtual environment created for stable-diffusion-webui-ux at C:\Users\shogz\loramaker\a1111ux\venv
Exception in thread Thread-186 (run_project_func):
Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "seait.py", line 99, in run_project_func
  File "util\projects_functions.py", line 108, in install
  File "subprocess.py", line 501, in run
  File "subprocess.py", line 969, in __init__
  File "subprocess.py", line 1438, in _execute_child
FileNotFoundError: [WinError 2] The system cannot find the file specified
```
Context: I had a working A1111 install (not via SEAIT) which, after an update attempt, ceased function. I tried updating torch, pip, and python manually, but to no avail. So it's probably related.

TIA!

**Comment by diStyApps** [comment]
It does seems to be related to Seait, 
If you try run webui-user.bat normally in automatic 1111 or any other forks what happens?
In this case seait.exe should be like this C:\Users\shogz\loramaker\seait.exe or how you have set your installation path via Seait
If automatic 1111 will not launch correctly delete the venv folder manual or via Seait, do not Uninstall, it will delete everything.
Update me.

i am currently working on better installation process for Seait.

**Comment by shoggot** [comment]
Okay. So:
```
exit code: 1

stderr:
C:\Program Files\Inkscape\bin\python.exe: No module named pip

Launch unsuccessful. Exiting.
Press any key to continue . . .
```

This, then, to me, isn't a SEAIT problem; it's a path problem. I'd re-installed python shortly before all of this, and I now suspect that it didn't set the path properly (just checked; by default it doesn't add to the path, unless you tick a box under 'advanced options' on the installer). So I've done that now.

Currently, webui-user.bat is churning through, doing its thing; when it's done, I'll delete it and try again via SEAIT and report back if there's still any problem, but I suspect it's likely fine now.

I guess that means my only suggestion for enhancement would be to check for valid path & pip functionality?
